settings.example.php

```diff
@@ -3,6 +3,8 @@
 
 	$settings = [
 		'site_title' => 'FlashPaper :: Self-Destructing Message',
+		'site_logo' => 'img/logo.png',
+		'display_title' => 'false', # true/false  
```

Comment:
It looks like the two new settings you added might be missing from the `docker/entrypoint.sh` and `docker/settings.php.TEMPLATE` files, so people who use Docker won't get the new settings

Reply:
Added settings to both files in [8171400](https://github.com/AndrewPaglusch/FlashPaper/pull/159/commits/8171400912bc3ba6b3135d2e04231c90ee80b1bb)

Updated docker/entrypoint.sh to no logo when no url is supplied in [713b3f5](https://github.com/AndrewPaglusch/FlashPaper/pull/159/commits/713b3f5569fae38b6cc3b70f5b682e79b5ac89ec)